examples/09-mesh_operations/00-create_polygons_and_polyhedrons.py

```diff
@@ -1,108 +1,474 @@
+    [5, 12],        # Line[Node] 31
+]
+
+"""SolidOnly Face[nodes]"""
+
+polyhedron_faces_node_connectivity = [
+    [15, 17, 18, 13, 8],        # Face[Node] 0
+    [17, 18, 13, 7],            # Face[Node] 1
+    [17, 18, 14, 1],            # Face[Node] 2
+    [16, 17, 18, 14, 2],        # Face[Node] 3
+    [17, 16, 0],                # Face[Node] 4
+    [17, 15, 4],                # Face[Node] 5
+    [16, 17, 15, 5],            # Face[Node] 6
+    [14, 18, 13, 10],           # Face[Node] 7
+    [2, 16, 0, 3],              # Face[Node] 8
+    [3, 1, 14, 2],              # Face[Node] 9
+    [11, 1, 17, 7],             # Face[Node] 10
+    [6, 0, 16, 5],              # Face[Node] 11
+    [5, 15, 4, 6],              # Face[Node] 12
+    [3, 0, 17, 1],              # Face[Node] 13
+    [9, 4, 15, 8],              # Face[Node] 14
+    [8, 13, 7, 9],              # Face[Node] 15
+    [7, 17, 4, 9],              # Face[Node] 16
+    [10, 14, 1, 11],            # Face[Node] 17
+    [11, 7, 13, 10],            # Face[Node] 18
+    [4, 17, 0, 6],              # Face[Node] 19
+    [12, 2, 14, 10],            # Face[Node] 20
+    [10, 13, 8, 12],            # Face[Node] 21
+    [8, 15, 5, 12],             # Face[Node] 22
+    [5, 16, 2, 12],             # Face[Node] 23
+]
+
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+"""CELL_FACE CONNECTIVITY"""
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+"""ShellOnly Face[lines]"""
+
+polygon_element_faces_connectivity = [
+    [0, 1, 2, 3],       # Face[Line] 0
+    [4, 5, 6, 3],       # Face[Line] 1
+    [7, 8, 9, 10],      # Face[Line] 2
+    [11, 1, 12, 13],    # Face[Line] 3
+    [14, 15, 16, 13],   # Face[Line] 4
+    [2, 17, 8, 4],      # Face[Line] 5
+    [18, 15, 19, 20],   # Face[Line] 6
+    [21, 22, 23, 20],   # Face[Line] 7
+    [10, 24, 18, 23],   # Face[Line] 8
+    [25, 5, 7, 26],     # Face[Line] 9
+    [10, 22, 27, 26],   # Face[Line] 10
+    [24, 21, 12, 18],   # Face[Line] 11
+    [28, 6, 25, 29],    # Face[Line] 12
+    [27, 21, 30, 29],   # Face[Line] 13
+    [19, 14, 31, 30],   # Face[Line] 14
+    [12, 0, 28, 31],    # Face[Line] 15
+]
+
+"""SolidOnly Volume[Face]"""
+
+polyhedron_element_faces_connectivity = [
+    [3, 7, 20, 21, 22, 23, 0, 6],   # Volume[Face] 0
+    [2, 8, 9, 13, 3, 4],            # Volume[Face] 1
+    [0, 5, 14, 15, 16, 1],          # Volume[Face] 2
+    [4, 6, 11, 12, 19, 5],          # Volume[Face] 3
+    [1, 10, 17, 18, 2, 7],          # Volume[Face] 4
+]
 
-# Add the polygon element to the MeshedRegion
-mesh.elements.add_shell_element(id=1, connectivity=connectivity)
 
 ###############################################################################
-# Define the polyhedron's connectivity
-
-# Define the faces connectivity
-polyhedron_faces_connectivity = [
-    [5, 6, 7, 8, 9],
-    [5, 6, 11, 10],
-    [5, 9, 14, 10],
-    [9, 14, 13, 8],
-    [8, 13, 12, 7],
-    [7, 12, 11, 6],
-    [10, 11, 12, 13, 14],
+"""
+Now also have to define a PropertyField relative the element_face reverse
```

Comment:
I don't think this sentence is clear enough. I would rephrase it to something like:

In addition to the relative previous `PropertyFields`, it is necessary to define the orientation of the faces relative to the cells that they surround. A given face has its nodal connectivity defined in a certain order, which means that the two cells that it bounds are either at the positive or negative sides of the face. We indicate this with the `elements_faces_reversed` PropertField, whose values can be either 0 or 1. By convention, 0 means that the face->node connectivity implies that the given cell is at side 0 of the face, and 1 the opposite.